docs/core/tools/dotnet-new.md

```diff
@@ -508,18 +738,26 @@ Create a .NET Standard class library project in the specified directory (availab
 
 `dotnet new classlib -lang VB -o MyLibrary`
 
-Create a new ASP.NET Core C# MVC application project in the current directory with no authentication:
+Create a new ASP.NET Core C# MVC project in the current directory with no authentication:
 
 `dotnet new mvc -au None`
 
-Create a new xUnit application:
+Create a new xUnit project:
 
 `dotnet new xunit`
 
 List all templates available for MVC:
 
 `dotnet new mvc -l`
 
+List all templates matching the *we* substring. No exact match is found, so substring matching runs against both the short name and name columns.
+
+`dotnet new we`
+
+List all templates matching the *ng* substring. Only one match is found, so the project is created.
+
+`dotnet new ng`
```

Comment:
Same thing here. Text might want to be something along the lines of (this probably needs work): 
"Attempt to invoke the template matching 'ng'. If a single match cannot be determined, list the template that are partial matches"